src/flapjackfeeder.c

```diff
@@ -220,6 +222,36 @@ int npcdmod_handle_data(int event_type, void *data) {
 
             host = find_host(hostchkdata->host_name);
 
+            customvariablesmember *currentcustomvar = host->custom_variables;
+            long initial_failure_delay = 0;
+            long repeat_failure_delay  = 0;
+            char *cur = temp_buffer, * const end = temp_buffer + sizeof temp_buffer;
+            temp_buffer[0] = '\x0';
+            while (currentcustomvar != NULL) {
+                if (strcmp(currentcustomvar->variable_name, "TAG") == 0) {
+                  cur += snprintf(cur, end - cur,
+                      "\"%s\",",
+                      currentcustomvar->variable_value
+                      );
+                }
+                else if (strcmp(currentcustomvar->variable_name, "INITIAL_FAILURE_DELAY") == 0) {
+                      initial_failure_delay = strtol(currentcustomvar->variable_value,NULL,10);
+                }
+                else if (strcmp(currentcustomvar->variable_name, "REPEAT_FAILURE_DELAY") == 0) {
+                      repeat_failure_delay = strtol(currentcustomvar->variable_value,NULL,10);
+                }
+                currentcustomvar = currentcustomvar->next;
+            }
+            cur--;
+            if (strcmp(cur, ",") == 0) {
+                cur[0] = '\x0';
+            }
+            else {
+                cur++;
+            }
+
```

Comment:
I'm quite confident that those lines are safe due to the test for NULL in line 230.
And my test config, that has definitions without customers in it, runs without any problem here.